DuggaSys/microservices/monolithic-to-microservices.md

```diff
@@ -0,0 +1,123 @@
+
+Disclaimer, this guide serves as a guideline and can not be followed 100% since the structure of all files differ. It's recommended to test after each step which include code changing to so its faster to spot if something goes wrong. When this guide was written no service using retrive information was used, write guide at step 5.
+***
+##Setting up file & folder:
+***
+
+###Step 1
+Locate the folder microservices:
+***DuggaSys -> microservices***
+
+###Step 2
+If a folder named after the monolithic service file exists skip to step 3.
+Else if it does not exist, create a folder named after the service file, for example codeviewerService.php(monolithic) becomes codeviewerService(Folder).
+
+###Step 3
+Create a file with name following the naming convention in Potential_micro_services:
+***Backend-models -> microservices -> duggaSys services***
+
+**Example:**
+| codeviewerService |
+| --- |
+| editBoxTitle_ms.php |
+| createBox_ms.php |
+
+***
+##Converting monolithic to microservice:
+###Step 1
+Copy the entire service file content from the monolithic php to your new made, the purpose of this is to simmer down 
+
+###Step 2
+Remove all the “opt”s that are not relevant to you. If your function is an else if, make it into an If statement. 
```

Comment:
Could be clearer, not entirely sure what this means. That means that someone further down will probably have a harder time understanding. Maybe add an example that shows what removing the "opt"s means?

Related to previous comment, this step could be different if you choose to change step 1. Either way, displaying an example of the simmer down approach would probably help clarifying.